### PR TITLE
Add Ruby gem scaffolding, update .gitignore

### PR DESCRIPTION
**Pull request by shelbyspees**
Used `bundle gem` to generate and update package files:
- /lib/revolution/ source file stubs
- /spec/ documentation stubs
- Rakefile
- Gemfile
- .travis.yml

Updated .gitignore with `bundle gem` recommendations.

For more information see: https://bundler.io/v1.13/guides/creating_gem

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,8 +1,15 @@
 ---
 dist: trusty
+sudo: false
```
Comment (shelbyspees) [inline .travis.yml]: Do we need this line?

Comment (smaslennikov) [reply .travis.yml]: yes

File: Gemfile [hunk 2]
```diff
@@ -1,4 +1,7 @@
 # frozen_string_literal: true
 source "https://rubygems.org"
 
-# gem "rails"
+git_source(:github) {|revolution| "https://github.com/grindrlabs/revolution" }
```
Comment (smaslennikov) [inline Gemfile]: I'm not clear on why this needs to be here

Comment (shelbyspees) [reply Gemfile]: It's generated by `bundle gem`, here's an explanation: https://stackoverflow.com/questions/41454333/meaning-of-new-block-git-sourcegithub-in-gemfile

**Pull request merged**